DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/merge_sort.hpp

```diff
@@ -163,7 +165,7 @@ std::vector<event> SubmitMergeSort(queue& q, size_t count, ValueT* buf_0,
   ValueT* buf[2] = {buf_0, buf_1};
 
   // using double buffering, so track the current buffer and have a simple
-  // lamda to compute the next buffer index
+  // lambda to compute the next buffer index
```

Comment:
I'm embarassed that I reviewed so much of your code without realizing that you don't know how to spell 'lambda' :-(